### PR TITLE
LL-842 receive flow - share address

### PR DESCRIPTION
**Pull request by valpinkman**
### Type

Feat

### Context

LL-842

### Parts of the app affected / Test plan

Receive Flow

### CODE REVIEW

File: src/components/ShareLink.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import React, { PureComponent } from "react";
+import { StyleSheet, Share } from "react-native";
+import Icon from "react-native-vector-icons/dist/Feather";
+import Touchable from "./Touchable";
+import LText from "./LText";
+import colors from "../colors";
+
+type Props = {
+  style?: *,
+  children: string | React$Element<*>,
```
Comment (gre) [inline src/components/ShareLink.js]: React$Node

File: src/components/ShareLink.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  string: string, // String to be shared
```
Comment (gre) [inline src/components/ShareLink.js]: chose a better name than `string` for the prop, maybe `value`

File: src/components/ShareLink.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    });
+  };
+
+  render() {
+    const { style, children } = this.props;
+
+    return (
+      <Touchable
+        event="ShareLink"
+        style={[styles.linkContainer, style]}
```
Comment (gre) [inline src/components/ShareLink.js]: remove the `style` prop. it's not needed from the user side so no need to add this complexity

can just `style={styles.linkContainer}`

File: src/components/ShareLink.js [hunk 4]
```diff
@@ -8,26 +8,25 @@ import LText from "./LText";
 import colors from "../colors";
 
 type Props = {
-  style?: *,
-  children: string | React$Element<*>,
-  string: string, // String to be shared
+  children: string | React$Node,
```
Comment (gre) [inline src/components/ShareLink.js]: React$Node includes `string` in it so you can just set the whole type as `React$Node`

Comment (valpinkman) [reply src/components/ShareLink.js]: shit... will change

**Pull request merged**